File: ext/nim/examples/strcat.nim
## ext/nim/examples/strcat.nim —— 与 lua / go / V 那三份 strcat **同一件事**
##
## 期望输出逐行相同：ab / hi there。
## nim 的串接是 `&`（数的 `+` 与它是两个算符）—— 映射的算符表里 `&` 直接映到 `concat`
## 那格内建，所以图上与 lua 的 `..`、go 的 `+` 落在同一处。

echo "a" & "b"
let s = "hi"
echo s & " " & "there"

File: ext/go/examples/strcat.go
// ext/go/examples/strcat.go —— 与 lua / nim / V 那三份 strcat **同一件事**
//
// 期望输出逐行相同：ab / hi there。
// go 用 `+` 接串（与数的 `+` 同一个算符）—— 落到的那一格由**操作数的种类**定：
// 两边是串就是串接。图上不为它开新节点（内建表里 `+` 与 `concat` 各一行）。

package main

import "fmt"

func main() {
	fmt.Println("a" + "b")
	s := "hi"
	fmt.Println(s + " " + "there")
}
